starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,63 @@
+@import url("https://fonts.googleapis.com/css?family=Open+Sans:300,600");
+
+body {
+    font-family: "Open Sans",Helvetica,Arial,sans-serif;
+    font-size:11pt;
+}
+
+.recipeTitle {
+    margin: auto;
+    display: block;
+    text-align: center;
+    font-size: 70pt;
+    font-weight: normal;
+    font-family:Helvetica, Arial;
+    color: #ffffff;
+    }
+
+#myTitle {
+    background:url("../images/apple-pie.jpg");
+    background-size: cover;
+    padding-top:350px;
+    padding-bottom:350px;
+    margin-bottom:30pt;
```

Comment:
It's always better to use the same unit for a web page, you could have stick to `px`.
We will see more about this when we will talk about responsive web pages.